src/back/spv/writer.rs

```diff
@@ -732,6 +773,36 @@ impl Writer {
     /// Primitive Instructions
     ///
 
+    fn parse_type_declaration_local(&mut self, local_ty: LocalType) -> (Instruction, Word) {
```

Comment:
we shouldn't have the "parse" in the names, since the backend isn't supposed to parse anything :)
consider "write" or "convert", or something along the lines

Reply:
Completely agreed, `write` it is, as we want to have the same terminology as the metal back-end 👍